guide/important-capabilities-dart.rst

```diff
@@ -144,10 +144,9 @@ Data assimilation algorithms available in DART
 DART allows users to test the impact of using multiple different types of
 algorithms for filtering, inflation/deflation, and covariance localization.
 
-DART offers numerous **filter algorithms**. These determine how the posterior
+DART offers numerous **filter algorithms** in the :ref:`QCEFF`. These determine how the posterior
```

Comment:
I believe "posterior" should be "prior".  The prior is updated by the algorithm to yield the posterior.   And then remove "and the prior ensemble"

Reply:
Thanks for checking this Kevin. changed to your suggestion.  Better?

Reply:
I see the fix in "Commits", but not in "Files Changed".
The fix looks good.